### ISSUE TITLE
udp proxy

### ISSUE THREAD

**Issue by lihui0213** [openresty/openresty#165]
in the last version of nginx 1.9.14,it has been supported udp proxy,there are any plans to update the nginx to 1.9.14 to supported tcp/udp proxy。

**Comment by agentzh** [comment]
@lihui0213 Yes, the next OpenResty release will probably be based on NGINX 1.9.14 or later. We're still doing extensive compatibility testing on our side.

**Comment by lihui0213** [comment]
thanks，I known。The new version of the plan released at what time？

**Comment by agentzh** [comment]
@lihui0213 Probably in the next few weeks if no other things pop up.

**Comment by lihui0213** [comment]
the nginx on windows is not suppored udp proxy,are there any plans to support the udp proxy on windows?

**Comment by agentzh** [comment]
@lihui0213 Are you talking about the official nginx windows binary or OpenResty's windows binary?

**Comment by lihui0213** [comment]
Yes,the code is only achieve the linux version ,is not support in windows.
In http://nginx.org/en/docs/windows.html  say three Known issues. The UDP proxy functionality is not supported.
I check the code ,it is in !win_32,the windows is not support recvmsg.so I think to ask there are any plans to fix this issues?

**Comment by agentzh** [comment]
@lihui0213 OpenResty is not an NGINX core. So generally speaking you should really ask the official NGINX team to "fix it" :)

We don't mind including a temporary NGINX core patch in OpenResty's source tree if it's good enough. So patches welcome :)
